include/solvers/mpm_base.tcc

```diff
@@ -81,8 +81,9 @@ mpm::MPMBase<Tdim>::MPMBase(const std::shared_ptr<IO>& io) : mpm::MPM(io) {
   }
 
   // Default VTK attributes
-  std::vector<std::string> vtk = {"velocities", "stresses", "strains",
-                                  "displacements"};
+  std::vector<std::string> vtk = {"velocities",     "stresses",
```

Comment:
In 3D will this output twice shear stresses and strains @ezrayst? Or the normal "stresses" and "strains" only output the normal components?

Reply:
Normal only

Reply:
@ezrayst `stress()` returns a 6-component vector, so Nanda is right

Reply:
@kks32 but `mesh` will only take the first 3 components right, so normal only.

Reply:
These are the line of codes.
https://github.com/cb-geo/mpm/blob/38172eed4eae47c23503661f3189b3684dba9611/include/mesh.tcc#L940-L947
I think @ezrayst  is right @kks32. `pdata` is a 6-component vector, but it is copied to `data` which is only 3-component. I checked that vtk accepts a 6-component vector too. So we might want to think about whether we should expand stresses or strains-like vectors to 6 component vector if we like to.

Reply:
Yes, it's only when you copy you truncate the components